Tensile/SolutionStructs.py

```diff
@@ -1167,9 +1169,15 @@ def __str__(self):
     if self["ComplexConjugateB"]:
       name += "C"
 
-    # precision and other
+    # DataTypes
+    # Format A/B/C/D/Alpha/Beta consistent with rocblas
+    name += "_"
+    name += self["DataType"].toChar()        # Type of A/B
+    name += self["DestDataType"].toChar()    # Type of C/D
+    name += self["ComputeDataType"].toChar() # Type of Alpha/Beta
+
```

Comment:
I agree the renaming of the kernel. But will this have some problems to merge the new inc. logics with existing logics in rocBLAS? I believe <H,S,S> and <B,S,S> may not have problems since no existing logics in rocBLAS. But what about our current <B,B,S>? The file names and kernel names are changed so maybe we need to do some tests on this ?

Reply:
For example, 
H,H,H+HPA: old name= _HBH_  <--> new name= _HHH_BH_
B,B,S+HPA: old name= _BBH_  <--> new name= _BBS_BH_ , 
We need to merge two different filenames. 

In addition, if a problem doesn't update the best solution (which is using new name), it might keep the old kernel which is using old name. I'm not sure if this will cause problems, maybe not, but do we need to make it more consistent ?

Reply:
@solaslin Transitioning existing names to the new naming scheme is not absolutely cast to stone at this point.   I'm not quite convinced that existing names must or should be mapped to the new names.  I'm not seeing any major benefit other than "consistency in naming".  All details must be carefully considered and assessed before a final technical decision can be made.

Reply:
@zaliu ok, I got it. but despite of the kernel names inside logic files, the merge script still need to be able to merge files with different filenames, or the new logic won't match the original filenames. I think this can be extended in #1177 @rosenrodt

Reply:
Or, what we can do is:
If (Ti != To)
   Use new ProblemType_<Ti/To/Tc>_<H/B> format
else
   Use ProblemType_<Ti/H/B> format as before

Reply:
To avoid the merging issue for now, I've restricted the kernel naming to be applied only on HSS and BSS kernels. This will be fine until (if/when) we decide to apply to all kernels. I am running short on time for the PR so I will only consider the specific cases, @zaliu @solaslin